scripts/install-wad.mjs

```diff
@@ -100,26 +112,55 @@ function selectRelease(releases, version) {
   return dstRelease;
 }
 
+/**
+ *
+ * @param {ReleaseInfo} release
+ * @returns {ReleaseAsset}
+ */
+function selectAsset(release) {
+  if (_.isEmpty(release.assets)) {
+    throw new Error(`WinAppDriver v${release.version} does not contain any matching releases`);
+  }
+  if (release.assets.length === 1) {
+    return release.assets[0];
+  }
+  // Since v 1.2.99 installers for multiple OS architectures are provided
+  for (const asset of release.assets) {
+    if (_.includes(asset.name, `win-${process.arch}.`)) {
```

Comment:
The existing code has additional mapping for architectures (specifically, `process.arch` `x32` -> release `x86`). Not sure if that needs to be added here?

Reply:
I deliberately ignored x86 as node.js is only built for x64 arch

Reply:
Isn't ia32 the same as 32-bit x86? Not actually sure what Node's `process.arch` returns on a 32-bit Windows.

Reply:
Added the mapping. I thought ia32 abbreviates Itanium rather than x86.